.editorconfig

```diff
@@ -7,10 +7,11 @@ root = true
 # A newline ending every file
 # Use 4 spaces as indentation
 [*]
-insert_final_newline = true
-indent_style = space
-indent_size = 4
+insert_final_newline     = true
```

Comment:
Just when I said I don't want to be a stylecop.. 🙈 Please don't do this weird aligning 😄

Reply:
Hi. Sorry for the spaces. It's actually due to an extension that I'm using for editorconfig file management that does that automatically, I didn't even noticed ha ha ha

About the "InstallHelper", are you referring to my other PR for the connection string in the web.config ? Because I don't think I changed this file for this PR. Moreover, there is something a bit special with editorconfig file ^^ If you set it to require curly braces and you don't put them, hitting Ctrl + K + D will add them. However, if you set it not to require curly braces... It won't remove the unnecessary ones, so that would be a manuel job.

Finally, for the webinfo file, I indeed simply open umbraco from the solution as a file system project.

Reply:
😁 Well maybe this is some kind of convention I don't know about, I see some of that weird aligning going on in the v8 version as well, so just leave it.

As for `InstallHelper` .. well.. scroll down... 😉 https://github.com/umbraco/Umbraco-CMS/pull/3795/files 

Anyway, no it should never remove existing curlies automatically, so that's all good!

Reply:
Oops... I must have forgot to switch branch or whatever... The change in the InstallHelper actually relates to https://github.com/umbraco/Umbraco-CMS/pull/3754. What's the best course of action ? Maybe it would be easier to cancel this PR and I do a new one with only the editorconfig file ?

Sorry for that, I'm still new in all this PR thingy :-D